### ISSUE TITLE
Feature: load energy per serve from openfoodfacts

### ISSUE THREAD

**Issue by CloCkWeRX** [patzly/grocy-android#538]
Grocy allows you to specify energy (kj).
Not clear if its total or serving size

1. Scan product to create
2. Get a result with nutrition facts modelled

Expected:
Recommends energy value from per serving? Or per 100g?

**Comment by MrJohnZoidberg** [comment]
Our app doesn't load the energy yet... Only the product name comes from OFF if the barcode was found.

**Comment by dominiczedler** [comment]
Maybe image can be loaded too...

**Comment by rugk** [comment]
I agree loading the image would be the best benefit IMHO, because it's so much easier recognizing stuff with an image.

**Comment by rugk** [comment]
BTW at least the image part may be solved with https://github.com/patzly/grocy-android/issues/844 as grocy seems to load that. (in contrast to calories, see https://github.com/grocy/grocy/issues/2649)
